meta/runtime.yml

```
@@ -318,3 +318,19 @@ plugin_routing:
         removal_version: 2.0.0
         warning_text: The onyx_wjh module has been moved to the mellanox.onyx collection
       redirect: mellanox.onyx.onyx_wjh
+  cliconf:
+    onyx:
+      redirect: mellanox.onyx.onyx
```

```suggestion
      deprecation:
        removal_version: 2.0.0
        warning_text: The onyx cliconf plugin has been moved to the mellanox.onyx collection
      redirect: mellanox.onyx.onyx
```
Same for the others.

---

fixed